pyproject.toml

```diff
@@ -1,18 +1,14 @@
-[tool.poetry]
+[project]
 name = "tuleap-documentation-fr"
 version = "0.0.0"
 description = ""
-authors = ["Enalean team"]
-license = "GPL-2.0-or-later"
+authors = [{ name = "Enalean team" }]
+requires-python = ">=3.12"
+dependencies = [
+    "Sphinx>=7.1.2,<8",
+    "sphinx-rtd-theme>=3.0.2,<4",
+    "sphinxcontrib-jquery>=4.1,<5",
+]
 
```

Comment:
question: in the FR repo, we don't need this block ?

```
[dependency-groups]
dev = ["sphinx-autobuild>=2021.3.14,<2022"]
```

Reply:
We do not have a watch target in this repo so it should not be needed.

Reply:
Ok, thanks